src/Raven.Server/Documents/ETL/Providers/SQL/RelationalWriters/RelationalDatabaseWriter.cs

```diff
@@ -229,23 +230,27 @@ private int InsertItems(string tableName, string pkName, List<ToSqlItem> toInser
                     }
                     catch (Exception e)
                     {
-                        if (_logger.IsInfoEnabled)
+                        if (token.IsCancellationRequested == false)
```

Comment:
maybe instead add another 'catch (OperationCanceledException)` and just do no-op there?